MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationAnimatorCoordinator.java

```diff
@@ -169,10 +169,14 @@ void feedNewAccuracyRadius(float targetAccuracyRadius, boolean noAnimation) {
     }
 
     float previousAccuracyRadius = getPreviousAccuracyRadius();
-    updateAccuracyAnimators(targetAccuracyRadius, previousAccuracyRadius);
-    playAnimators(
-      noAnimation || !accuracyAnimationEnabled ? 0 : ACCURACY_RADIUS_ANIMATION_DURATION,
-      ANIMATOR_LAYER_ACCURACY);
+
+    //Draw radius only if necessary
+    if (targetAccuracyRadius != 0.0 || targetAccuracyRadius != previousAccuracyRadius) {
```

Comment:
Why are you discarding `0.0` as an invalid update?

Reply:
updated